build-system/azure-pipeline.template.yaml

```diff
@@ -1,12 +1,14 @@
 parameters:
   name: ''
+  displayName: ''
```

Comment:
@Arkatufus ah, looks like the build system changes blew up this PR - could you revert these?

Reply:
Just the Azure pipelines file changes - I'll have to go through and troubleshoot this in Azure DevOps otherwise. This project is using a slightly older setup.

Reply:
Done, hope that does the trick.